src/scandeval/benchmark_dataset.py

```diff
@@ -468,6 +483,23 @@ def _load_prepared_data(
                             keep_in_memory=True,
                         )
 
+                        # Determine if we should strip the prompts. This is the case if
+                        # the tokenizer needs to include the space as part of the label
+                        # token
+                        labels_to_be_generated = list(
+                            self.dataset_config.prompt_label_mapping.values()
+                        )
+                        strip_prompts = should_prompts_be_stripped(
+                            labels_to_be_generated=labels_to_be_generated,
+                            tokenizer=tokenizer,
+                        )
+                        if strip_prompts:
+                            test = test.map(
```

Comment:
I am unsure what test is here - I would rename it "prompt_dataset" (assuming that is what it is).

Reply:
It's the test split :) 

Prompt dataset is misleading, since this both for decoders and encoders

Reply:
Ah right - then that is my misunderstanding keep it as is.